jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -26,6 +26,7 @@
 #include "ecma-objects-arguments.h"
 #include "ecma-try-catch-macro.h"
 #include "jcontext.h"
+#include "jerryscript-core.h"
```

Comment:
but the whole point of the rewrite is to completely get rid of the jerry_ types in internals of the library, isn't it?